index.html

```diff
@@ -9,203 +8,189 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@300;400;500&display=swap" rel="stylesheet"> 
     <link rel="stylesheet" href="./CSS/Styles.css">
```

Comment:
- [ ] **Important**: I just noticed that your project doesn't match any style because you gave the wrong name to the `styling file`. You wrote `href="./CSS/Styles.css"` with `S`(Styles.css) in capital letter instead of ``href="./CSS/styles.css"`` with `s`(styles.css) in lowercase. For good practices, I suggest you every time test your project before making any `Request review`, this will help you to avoid errors like that.